src/qt/mainwindow.cpp

```diff
@@ -65,13 +65,14 @@ MainWindow::MainWindow(
     m_pubsub->setHorizontalHeaderItem(7, new QStandardItem(QString("Ownership")));
     m_pubsub->setHorizontalHeaderItem(8, new QStandardItem(QString("Durability")));
     m_pubsub->setHorizontalHeaderItem(9, new QStandardItem(QString("Liveliness")));
-
+    m_pubsub->setHorizontalHeaderItem(10, new QStandardItem(QString("Data Representation")));
```

Comment:
I think we could manage to reduce wide columns if we shorten this text
```suggestion
    m_pubsub->setHorizontalHeaderItem(10, new QStandardItem(QString("Data Repr.")));
```